src/Psalm/Internal/Cli/Psalter.php

```diff
@@ -257,9 +261,13 @@ public static function run(array $argv): void
         }
 
         $debug = array_key_exists('debug', $options) || array_key_exists('debug-by-line', $options);
-        $progress = $debug
-            ? new DebugProgress()
-            : new DefaultProgress();
+        if ($debug) {
+            $progress = new DebugProgress();
+        } elseif (isset($options['no-progress'])) {
+            $progress = new VoidProgress();
```

Comment:
you seem to be missing an import for VoidProgress here

Reply:
I'm not sure it's classified as Code Style issue given it was an outright bug (Can php cs even know this class does not exists in this namespace??)